spinta/backends/postgresql/types/geometry/encode.py

```diff
@@ -40,8 +46,21 @@ def prepare_dtype_for_response(
     action: Action,
     select: dict = None,
 ):
-    shape = to_shape(value)
+    if value.srid not in [WGS84, -1]:
+        wgs_crs = CRS(f'EPSG:{WGS84}')
+        value_crs = CRS(f'EPSG:{value.srid}')
+        project = Transformer.from_crs(crs_from=value_crs, crs_to=wgs_crs, always_xy=True).transform
+        shape = transform(project, to_shape(value))
```

Comment:
Transformed value should only be used in the URL to OpenStreetMap, because OpenStreetMap uses WGS, but actual value (link text) should be shown in original projection.